src/rebar_erlc_compiler.erl

```diff
@@ -26,9 +26,9 @@
 %% -------------------------------------------------------------------
 -module(rebar_erlc_compiler).
 
--export([compile/1,
-         compile/3,
-         compile/4,
+-export([compile/1, compile/2, compile/3,
```

Comment:
i don't think so. i added it a month ago ([here][https://github.com/rebar/rebar3/commit/9d82215296205d2d30c152c7d7c6fcb8e4a92443]) for the ct provider but it no longer needs it with the changes to the compiler